### ISSUE TITLE
são carlos value of entity places sometimes comes without metadata

### ISSUE THREAD

**Issue by fczuardi** [fczuardi/calamarcopollo#56]
Here are two instances of the same statement causing wit to return different outcomes, from the logs:

```
Message: de são carlos pra são paulo
        de sao carlos pra sao paulo
outcome {"text":"de sao carlos pra sao paulo","entities":{"origin":[{"_body":"sao carlos","confidence":0.9998014023978596,"_start":3,"_end":13,"_role":"origin","_entity":"places","type":"value","value":"sao carlos"}],"destination":[{"_body":"sao paulo","confidence":0.9969396806613915,"_start":18,"_end":27,"_role":"destination","metadata":"{\"slugs\": [\"sao-paulo-sp-todos\", \"sao-paulo-tiete-sp\", \"sao-paulo-barra-funda-sp\", \"sao-paulo-jabaquara-sp\", \"aeroporto-congonhas-sp\"]}","_entity":"places","type":"value","value":"Sao Paulo, SP"}],"trip":[{"confidence":0.999999997430276,"_entity":"trip","value":"info"}]}}
chat, from, date { id: '1704285519822058',
  session: { destination: undefined, origin: undefined },
  date: 1466100093947 } { first_name: 'Gabriel',
  last_name: 'Soga',
  profile_pic: 'https://scontent.xx.fbcdn.net/v/t1.0-1/p200x200/10410755_10207995885728260_1621715811033008293_n.jpg?oh=b58d7b9e3a7342c245ca12547e6d60b1&oe=57D75DEE',
  locale: 'en_US',
  timezone: -3,
  gender: 'male' } 1466100093947
reply Desculpa, mas não conheço sao carlos 😶.
```

```
Message: de são carlos pra são paulo
        de sao carlos pra sao paulo
outcome {"text":"de sao carlos pra sao paulo","entities":{"origin":[{"_body":"sao carlos","confidence":0.9998014023978596,"_start":3,"_end":13,"_role":"origin","metadata":"{\"slugs\": [\"sao-carlos-sp\"]}","_entity":"places","type":"value","value":"Sao Carlos, SP"}],"destination":[{"_body":"sao paulo","confidence":0.9969396806613915,"_start":18,"_end":27,"_role":"destination","metadata":"{\"slugs\": [\"sao-paulo-sp-todos\", \"sao-paulo-tiete-sp\", \"sao-paulo-barra-funda-sp\", \"sao-paulo-jabaquara-sp\", \"aeroporto-congonhas-sp\"]}","_entity":"places","type":"value","value":"Sao Paulo, SP"}],"trip":[{"confidence":0.999999997430276,"_entity":"trip","value":"info"}]}}
chat, from, date { id: '1704285519822058',
  session:
   { destination: 'Sao Paulo, SP',
     origin: undefined,
     timeFilter: null,
     destinationMeta: { slugs: [Object] },
     originMeta: null },
  date: 1466100102129 } { first_name: 'Gabriel',
  last_name: 'Soga',
  profile_pic: 'https://scontent.xx.fbcdn.net/v/t1.0-1/p200x200/10410755_10207995885728260_1621715811033008293_n.jpg?oh=b58d7b9e3a7342c245ca12547e6d60b1&oe=57D75DEE',
  locale: 'en_US',
  timezone: -3,
  gender: 'male' } 1466100102129
requesting https://api.clickbus.com.br/api/v1/trips?from=sao-carlos-sp&to=sao-paulo-sp-todos&departure=2016-06-16
reply arrived
```

**Comment by fczuardi** [comment]
happened with a bridge tester as well:

![screen shot 2016-06-20 at 1 18 01 pm](https://cloud.githubusercontent.com/assets/7760/16201873/a642ccec-36e9-11e6-9bcc-5f99968c1005.png)

**Comment by fczuardi** [comment]
and another ![screen shot 2016-06-20 at 1 33 18 pm](https://cloud.githubusercontent.com/assets/7760/16202353/a84627e4-36eb-11e6-9a89-078206320fe2.png)

**Comment by fczuardi** [comment]
another:

![screen shot 2016-06-20 at 1 45 42 pm](https://cloud.githubusercontent.com/assets/7760/16202724/5d3e8e92-36ed-11e6-95fd-30effd79f75a.png)

**Comment by fczuardi** [comment]
another:

![screen shot 2016-06-20 at 1 51 14 pm](https://cloud.githubusercontent.com/assets/7760/16202895/15e3834e-36ee-11e6-8c82-c665cd6d4f7e.png)

**Comment by fczuardi** [comment]
another:

![screen shot 2016-06-20 at 1 57 16 pm](https://cloud.githubusercontent.com/assets/7760/16203092/f544c944-36ee-11e6-91aa-23d0835c263d.png)

**Comment by GabrielSoga** [comment]
Happened again on wit:
![w-o_metadata](https://cloud.githubusercontent.com/assets/19688935/16314816/078d83c0-3956-11e6-8c03-b20348005a7c.png)

**Comment by fczuardi** [comment]
while we wait for a wit bugfix, the workaround might be to fix #11 and have a local copy of all slugs that we know so we can rely less on entity values metadata

**Comment by fczuardi** [comment]
closing as fixed because we workaround this wit bug on our end
